app/controllers/users_controller.rb

```diff
@@ -17,6 +17,13 @@ def create
 
     if @user.save
 
+      begin
+        Stripe::Charge.create(card: params[:stripeToken], amount: 999, description: "Myflix charge for #{@user.email}", currency: 'usd') unless Rails.env.test?
```

Comment:
These types of checks should not be in your app code, generally your initializers should take care of whatever setup makes your code okay for all environments.